docs/localize.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Localize Your Connector
+---
+
+Tableau products have been localized to English (United States), English (United Kingdom), French, German, Brazilian Portuguese, Spanish, Korean, Japanese, simplified Chinese and traditional Chinese.
+
+Connector SDK supports localization of connectors to the languages mentioned above.
+
+The strings showing up on Tableau UI can be localized. For example, the connector name and the prompts in connection dialog.
+
+For the best user experience with Tableau, it is highly recommended that translation support is included.
+## Specify localized Strings
+Localized strings are specified using tag `@string/<string_id>/`, for example, `@string/database_prompt/`. You can use the tag anywhere a string is defined in `manifest.xml` or Tableau Custom Dialog (.tcd) file.
+
+Here is an example of a Tableau Custom Dialog (.tcd) file with localized strings.
+```xml
+<!-- Connection Dialog -->
+<connection-dialog class='sample'>
+  <connection-config>
+    <authentication-mode value='Basic' />
```

Comment:
To keep the sample snippets short and simple I've been using ..., I think you could replace all the auth stuff with ... here.